### DISCUSSION TITLE
Export trips to a geo file format

### DISCUSSION THREAD

**Discussion by jeyemwey** [Traewelling/traewelling#2025] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
As a user, I'd like to track my journeys via Träwelling, and later export my trips to other kinds of software, such as Geo-Spatial Software.

**Describe the solution you'd like**
Add an option to the exporting view, generate a file with all trips in a specified time frame. This file should contain:
* Journeys (origin, destination with date times) and additional data (status text? mode of transport? Operator?)
* Polyline data of the track, if available
* Interim stops

**Describe alternatives you've considered**
We need to research which file format is widely accepted and can contain the data. From my Geocaching days, I remember KML and GPX, but I have no idea if they're replaced by something newer/fancier/cooler/easier.

**Additional context**
* Inspired by [this thread](https://twitter.com/BehrPictures/status/1596868517756014592) that mentioned Träwelling on the side.

**Comment by MrKrisKrisu** [comment]
> We need to research which file format is widely accepted and can contain the data. 

I would prefer GeoJSON for this export.

**Comment by ShinyEmeraldGames** [comment]
> I would prefer GeoJSON for this export.

Are there any plans for this yet? Would be really cool to be able to export your journeys' GeoJSON data, with a journey identifier (primary key / UID) to be able map it to the rest of the journey data from e.g. csv journey exports. 
I know the data already exists in the API, because [this](https://fabian-c.github.io/Traewelling-Maps/docs) project already uses that to fetch the poly lines.
Since the API is not yet in a stable state and I'm working on a journey visualizer that uses the csv-exports as its data source. Would be really helpful to have the polyline as GeoJSON data for this reason.
